modes/parkour/interface.lua

```diff
@@ -531,8 +531,9 @@ onEvent("NewPlayer", function(player)
 end)
 
 onEvent("PlayerDataParsed", function(player, data)
-	if data.parkour.mort == 1 then
-		system.bindKeyboard(player, 77, true, true)
+	if data.parkour.mort then
+		system.bindKeyboard(player, data.parkour.mort == 1 and 77 or data.parkour.mort == 0 and 46, true, true)
```

Comment:
`data.parkour.mort == 1 and 77 or 46`

`data.parkour.mort` can be either 1 or 0, if it is not 1, we don't need to make another check if it is 0.